package.json

```diff
@@ -71,59 +32,18 @@
     "grapheme-splitter": "^1.0.2",
     "inquirer": "~3.3.0",
     "json-stringify-safe": "~5.0.1",
-    "mkdirp": "~0.5.1",
+    "mkdirp": "~1.0.4",
     "npm-install-package": "~2.1.0",
-    "optimist": "~0.6.1",
```

Comment:
Only used in the CLI and deprecated, upgrade path would have been long for a piece we don't use.